docs/tutorials/using-pixy-camera.md

```diff
@@ -0,0 +1,194 @@
+
+    pixy.mode = 'ALL'
+
+The Pixy camera has the following modes:
+* ALL: the camera searches for all signatures you've set for it.
+* SIGi: the camera searches for signature #i (i=1 to 7).
+
+The data which you can retreive from the camera depends on the camera mode. You
+can find detailed information on [this page](http://www.ev3dev.org/docs/sensors/charmed-labs-pixy-cmucam5-for-lego/).
+We will explain it to you with some examples.
+
+When mode is set to 'ALL' retreive data the following way:
+
+    sig = pixy.value(1)*256 + pixy.value(0) # Signature of largest object
+    x_centroid = pixy.value(2)    # X-centroid of largest SIG1-object
+    y_centroid = pixy.value(3)    # Y-centroid of largest SIG1-object
+    width = pixy.value(4)         # Width of the largest SIG1-object
+    height = pixy.value(5)        # Height of the largest SIG1-object
+
+When mode is set to one of the signatures (e.g. `SIG1`) retreive data
+as follows:
+
+    count = pixy.value(0)  # The number of objects that match signature 1
+    x = pixy.value(1)      # X-centroid of the largest SIG1-object
+    y = pixy.value(2)      # Y-centroid of the largest SIG1-object
+    w = pixy.value(3)      # Width of the largest SIG1-object
+    h = pixy.value(4)      # Height of the largest SIG1-object
+
+Below are two practical examples. They are tested with the latest version of ev3dev
+(as of 21 Dec 2016) and with CMUcam5 Pixy camera for LEGO Mindstorms.
+
+## Example 1 - Display detected object on EV3-LCD
+
+In this example the camera searches for objects with signature SIG1
+and displays the data graphically on the LCD of the EV3. It draws a 
+rectangle centered at the X and Y centroid coordinates with measured
+width and height. End this script by pressing Ctrl+C.
+
+```
+#!/usr/bin/env python3
+from ev3dev.ev3 import *
+
+lcd = Screen()
+
+# Connect Pixy camera
+pixy = Sensor(address='in1')
+assert pixy.connected, "Connecting PixyCam"
+
+# Set mode
+pixy.mode = 'SIG1'
+
+while True:
+  lcd.clear()
+  if pixy.value(0) != 0:  # Object with SIG1 detected
+    x = pixy.value(1) 
+    y = pixy.value(2)
+    w = pixy.value(3)
+    h = pixy.value(4)
+    dx = int(w/2)       # Half of the width of the rectangle
+    dy = int(h/2)       # Half of the height of the rectangle
+    xb = x + int(w/2)   # X-coordinate of bottom-right corner
+    yb = y - int(h/2)   # Y-coordinate of the bottom-right corner
+    lcd.draw.rectangle((xa,ya,xb,yb), fill='black')
+    lcd.update()
+```
+
+The video shows this script running:
+{% include /util/youtube-embed.html youtube_video_id="b2LZpY1qbKE" %}
+
+## Example 2 - Following an object
+
+This example uses a EV3 with Pixy camera, two LargeMotors and a TouchSensor.
+The robot follows the object with signature SIG1. To stop the program the
+user has to press the TouchSensor.
+This example uses a simple implementation for a PID-controller. The example
+works very well, but finetuning of the PID-constants can make the robot 
+react en move smoother.
```

Comment:
That word replacement wouldn't make the sentence understandable... I think "en move" can be removed, although other fixes are obviously possible.